sklearn/tests/test_kernel_approximation.py

```diff
@@ -215,6 +215,35 @@ def test_rbf_sampler():
     assert np.mean(error) <= 0.05  # mean is fairly close
 
 
+@pytest.mark.parametrize("dtype", [np.float32, np.float64])
+def test_skewed_chi2_sampler_fitted_attributes_dtype(dtype):
```

Comment:
Please add a small comment to explain the purpose of the test, as done in https://github.com/scikit-learn/scikit-learn/pull/24317